src/applications/simple-forms/21-10210/config/form.js

```diff
@@ -37,6 +38,26 @@ import testData from '../tests/e2e/fixtures/data/noStmtInfo.json';
 
 const mockData = testData.data;
 
+const pageFocus = () => {
+  return () => {
+    const { pathname } = document.location;
+    let focusSelector = '';
+
+    if (pathname.includes('claim-ownership')) {
+      focusSelector = '#main h1[data-testid="form-title"]';
+    } else if (pathname.includes('claimant-type')) {
+      focusSelector = '#main .schemaform-first-field legend';
+    } else if (!pathname.includes('review-and-submit')) {
+      // since form-level useCustomScrollAndFocus is true,
+      // this fn fires on every page change, so we need to
+      // provide default focusSelector for all other pages
+      focusSelector = defaultFocusSelector;
+    }
+
+    focusElement(focusSelector);
```

Comment:
Could simplify this by just returning the string if you want since it'll handle the scroll for string values in the [scroll utility](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/utilities/ui/index.js#L94). Maybe you already tried that though...

Reply:
Can't remove.  FormNav fires my custom-focus function on _every_ chapter once useCustomScrollAndFocus is enabled at the form-level, so FormNav's own catch-all's never reached, so we need to provide our own catchall in our custom-focus fn.

Reply:
Just wanted to avoid regression risks here, in case someone goes in a updates defaultFocusSelector and 'orphans' my catchall.